database.json

```diff
@@ -1521,6 +1521,12 @@
     "repo": "online",
     "desc": "Check if you're currently online and connected to the internet"
   },
+  "isdocker": {
```

Comment:
@yg The test is failing because this key is not in its right place (alphabetical order). Move it to after `isIP` and before `is_exe`, line 1137.

Reply:
Thanks @AnandChowdhary :rocket: